City-of-Helsinki/drupal-helfi-rekry
# UHF-7195 News feed

# [UHF-7195](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7195)


## What was done


* Installed and configured the `helfi_news_feed` module

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-7195_news_feed`
  * `make fresh`
* Run `make drush-cr`

## How to test


- Go to create landing page, basic page or TPR unit/service
- Check that you can add "News list" paragraph in to 
![image](https://user-images.githubusercontent.com/1712902/197695800-be823101-fb54-4394-bf24-d0707b53e389.png)

## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/383
